api/db/repository/user.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy.orm import Session
+from schemas.user import UserCreate, UserBase
+from db.models.user import User
+from core.hashing import Hasher
+from fastapi.encoders import jsonable_encoder
+
+
```

Comment:
**This is just a suggestion for the future**:
you can create a separate class:
```
class UserRepository:
    def __init__(self, db: Session):
         ...
```
or 
```
class UserRepository(AsyncClass):
   async  def __ainit__(self, db: Session):
         ...
```
this is usually made to avoid situations where you accidentally called create method for petition (for example)

Reply:
Okay, I will get a look at it